app/helpers/form_helper.rb

```diff
@@ -239,12 +239,12 @@ def counter_f(f, attr, options = {})
     react_form_input('counter', f, attr, options)
   end
 
-  def submit_or_cancel(f, overwrite = false, args = { })
+  def submit_or_cancel(f, overwrite = false, args = { }, button_text = "Submit")
```

Comment:
Even though it works, I'd suggest to stick with the current parameter order (optional parameters are before the hash parameter). But this could potentially break other places, so maybe we could put this within `args`? And in the body of the method just simply `button_text = args.delete(:button_text) || _('Submit')` .

Reply:
> Even though it works, I'd suggest to stick with the current parameter order (optional parameters are before the hash parameter). But this could potentially break other places, so maybe we could put this within `args`? And in the body of the method just simply `button_text = args.delete(:button_text) || _('Submit')` .

I tried to make it a bit more simple than creating new variable. I am not sure though, whether or not it is good for readability from your point of view.